src/components/Labels.tsx

```diff
@@ -77,84 +68,94 @@ export const Labels: FunctionComponent<Props> = ({
     updateAllLabels();
   };
 
-  const handleClick = () => {
-    setIsOpen(!isOpen);
-  };
-
   useEffect(() => {
     // Re-render assigned labels at change of active annotation ID.
     updateAllLabels();
   }, [activeAnnotationID]);
 
   return (
     <>
-      <List component="div" disablePadding>
+      <List component="div" disablePadding style={{ width: "100%" }}>
+        <ListItem>
+          <InputBase
+            placeholder="New label"
+            value={newLabel}
+            onChange={handleNewLabelChange}
+            inputProps={{
+              style: { fontSize: 18, marginRight: 57, marginLeft: -7 },
+            }}
+            startAdornment
+          />
+
+          <IconButton
+            type="submit"
+            aria-label="add-new-label"
+            onClick={handleAddLabel(newLabel)}
+            edge="end"
+          >
+            <SVG
+              src="./src/assets/add-icon.svg"
+              width="12px"
+              height="auto"
+              fill="#A1A1A1"
+            />
+          </IconButton>
+        </ListItem>
+        <Divider
+          style={{
```

Comment:
As above, let's remove in-line styling. I would suspect we should be using classes for our theme and ids for local 'with stylings'?

Reply:
we probably shouldn't even need IDs

Reply:
Possibly... But if we have no in-line stylings but only want to style a single, specific item in a very specific way I would, traditionally, use IDs? Purely for the priority of specification?

Reply:
Yea, my CSS linter bans them, and I very very rarely have to override it....but if we need them